es_sink/setup.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
```

Comment:
License headers ?

Reply:
AFAIK, this is fine. Let me check whether MIT works or I need to change to Apache

Reply:
MIT works - compatible w Apache 2